Sources/Guaka/Parsing/FlagSet+Parsing.swift

```diff
@@ -40,36 +40,35 @@ extension FlagSet {
 
       switch token {
 
-      case let .longFlagWithEqual(name, value):
+      case let .longFlagWithEqual(name, value),
+           let .shortFlagWithEqual(name, value):
```

Comment:
Since these both have the same arguments and do the exact same thing, I combined the cases to reduce the code duplication